front_end_pocket_poll/lib/widgets/members_page.dart

```diff
@@ -306,6 +308,12 @@ class _MembersPageState extends State<MembersPage> {
           "Cannot add $username because they have previously left this group.",
           this.context);
       hideKeyboard(this.context);
+    } else if (widget.displayedMembers.length >= Globals.maxGroupMembers) {
+      showErrorMessage(
+          "Error",
+          "Cannot add $username because the max group size of ${Globals.maxGroupMembers} has been met.",
```

Comment:
Small request but could you put the username in quotes? Makes it easier to read imo and we do that elsewhere in the app.

Reply:
I was actually staying consistent with the other code on this page that does the same thing. I agree with you though so I'll update this and the old code.